pkg/events/state.go

```diff
@@ -0,0 +1,46 @@
+package events
```

Comment:
Going to policy.

Reply:
Not possible. events.DefinitionGroup.GetTailCalls() makes use of EventsStates, so moving it to policy pkg causes import cycle.